install/install.sh

```
@@ -77,14 +79,23 @@ loveletter_service(){
 	sudo cp $(pwd)/Rpi_Crons/loveletter.service /etc/systemd/system/
 	sudo cp $(pwd)/Rpi_Crons/daily_restart.service /etc/systemd/system/
 	sudo cp $(pwd)/Rpi_Crons/daily_restart.timer /etc/systemd/system/
-	add_or_replace_variable "WorkingDirectory" "$(pwd)" "/etc/systemd/system/loveletter.service"
-	add_or_replace_variable "ExecStart" "$(pwd)/start.sh" "/etc/systemd/system/loveletter.service"
+	sudo cp $(pwd)/Rpi_Crons/loveletter_extraction.service /etc/systemd/system/
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)